kserve-modelmesh.yaml

```diff
@@ -1,11 +1,12 @@
 package:
   name: kserve-modelmesh
   version: 0.12.0
-  epoch: 2
+  epoch: 3
   description: The ModelMesh framework is a mature, general-purpose model serving management/routing layer designed for high-scale, high-density and frequently-changing model use cases.
   dependencies:
     runtime:
       - bash # entrypoint script uses bash
+      - linux-pam
```

Comment:
from a quick skim, it looked like the package needed linux-pam at runtime but looking more deeply into it that's not the case.

upstream is using ubi-minimal as base which comes with `/etc/environment` out of the box. This comes from `setup` package which configures the system settings at start.

```bash
$ docker run --rm -it registry.access.redhat.com/ubi8/ubi-minimal:latest
[root@0ec0def7e2db /]# rpm -qf /etc/environment
setup-2.12.2-9.el8.noarch
```

So, I think upstream `/etc/environment` is different considering it comes from `setup` than our so I'll be more comfortable creating `/etc/environment` using touch than shipping this package with linux-pam because linux-pam comes with more shared-libraries and auth related binaries.

I think the original changes were good enough for creating `/etc/environment`

TLDR is: we cannot pull in `/etc/environment` for this package because linux-pam is not a runtime dependency for this package and there's variation b/w who owns `/etc/environment` in our base image vs upstream base image. 

Sorry but for now it looks like a revert of https://github.com/wolfi-dev/os/pull/31831/commits/1f2788a87da76f9b0f1dc9876015bb67680a4b1c will be a better approach here.